### ISSUE TITLE
Create GitHub pages for this repo

### ISSUE THREAD

**Issue by TravorLZH** [Cynthia7979/Those-Web-Things#1]
Hey Cynthia!
You can now make GitHub generate HTML pages online by making some configurations, so everyone will be able to see your work.

**Comment by TravorLZH** [comment]
You can visit my page at [TRAVOR.TOP][1]

[1]: https://travor.top/labs.html

**Comment by Cynthia7979** [comment]
I’ve been heard of it before. Maybe I’ll try it once I got time. Your site is pretty cool by the way.

发自我的 iPhone

在 2019年6月19日，18:36，Travor Liu <notifications@github.com<mailto:notifications@github.com>> 写道：


Hey Cynthia!
You can now make GitHub generate HTML pages online by making some configurations, so everyone will be able to see your work.

**Comment by TravorLZH** [comment]
Yeah, but I have not updated the posts for a long time (the latest post 
was published last year!). Hope you can get your pages working as soon as 
possible

Travor Liu
